src/NuGet.Services.Metadata.Catalog.Monitoring/ValidationCollectorFactory.cs

```diff
@@ -42,63 +40,28 @@ public ValidationCollectorFactory(ILoggerFactory loggerFactory)
         }
 
         public Result Create(
```

Comment:
>Create [](start = 22, length = 6)

Could we add documentation for this method?

Reply:
added